### ISSUE TITLE
🐛Bug Report: Stopped working with html-loader v1.0.0

### ISSUE THREAD

**Issue by cb-eli** [YashdalfTheGray/auto-ngtemplate-loader#23]
**Describe the bug**
html-loader was updated to v1.0.0 with breaking changes. Since then ngtemplate-loader is breaking my build.

**Urgency**
⏰Normal

**To Reproduce**
1. Update html-loader to v1.0.0
2. Run webpack build

**Environment**
 - OS: Windows 10.0
 - Node.js version: v13.7.0
 - `webpack` version: 4.42.1
 - `auto-ngtemplate-loader` version: 2.1.0

**Additional context**
Webpack config:

```
        resolveLoader: {
            alias: {
                'ngtemplate': 'ngtemplate-loader?relativeTo=' + path.resolve(__dirname, CONSTS.SCRIPTS) + '/'
            }
        },
```

Error:

```
Uncaught Error: Module parse failed: Unexpected token (3:0)
File was processed with these loaders:
 * ./node_modules/ngtemplate-loader/index.js
 * ./node_modules/html-loader/dist/cjs.js
You may need an additional loader to handle the result of these loaders.
| var path = 'Application/Common/Templates/Directives/dropdown.html';
| var html = // Module
> var code = "<div>test</div>";
| // Exports
| module.exports = code;;
    at Object../node_modules/ngtemplate-loader/index.js?relativeTo=C:\Projects\project-name\Code\WebSite\project-name\Scripts/!./Scripts/Application/Common/Templates/Directives/dropdown.html (MyScript.js:1363)
    at __webpack_require__ (bootstrap:84)
    at Module../Scripts/Application/Pages/xxx/xxx/Directives/xxx.js (xxx.js:635)
    at __webpack_require__ (bootstrap:84)
    at Module../Scripts/Application/Pages/xxx/xxx/Directives/xxx.js (xxx.js:1)
    at __webpack_require__ (bootstrap:84)
    at Module../Scripts/Application/Pages/xxx/xxx/Directives/xxx.js (xxx.js:1)
    at __webpack_require__ (bootstrap:84)
    at Module../Scripts/Application/Pages/xxx/xxx/Directives/xxx.js (xxx.js:1)
    at __webpack_require__ (bootstrap:84)
```

**Comment by YashdalfTheGray** [comment]
Okay so I did some looking into it, it seems like [this code](https://github.com/WearyMonkey/ngtemplate-loader/blob/master/index.js#L48-L54) picks up the wrong thing after the update because it's probably looking for a very specific string to be where it's not. My current hypothesis is that the `html-loader` output changed in some way. Consequently, it makes [this code](https://github.com/WearyMonkey/ngtemplate-loader/blob/master/index.js#L48-L54) not work the way that it should which causes the error. 

At least for right now, until `ngtemplate-loader` is fixed, you can pin the version of `html-loader` to be "0.5.5" and that won't break the build.

**Comment by YashdalfTheGray** [comment]
I'm definitely not fully awake, I found [this issue](https://github.com/WearyMonkey/ngtemplate-loader/issues/85) in the `ngtemplate-loader` repo that you've opened that has gotten to a similar root cause as me. 😅 

Oh well. I learned something new.

**Comment by cb-eli** [comment]
> Okay so I did some looking into it, it seems like [this code](https://github.com/WearyMonkey/ngtemplate-loader/blob/master/index.js#L48-L54) picks up the wrong thing after the update because it's probably looking for a very specific string to be where it's not. My current hypothesis is that the `html-loader` output changed in some way. Consequently, it makes [this code](https://github.com/WearyMonkey/ngtemplate-loader/blob/master/index.js#L48-L54) not work the way that it should which causes the error.
> 
> At least for right now, until `ngtemplate-loader` is fixed, you can pin the version of `html-loader` to be "0.5.5" and that won't break the build.

Yes, for now I pinned to 0.5.5. Thanks for the response :)

**Comment by chiptus** [comment]
ngtemplate-loader should be released with a fix soon, no change needed in this plugin.

**Comment by YashdalfTheGray** [comment]
Ack!

**Comment by YashdalfTheGray** [comment]
https://www.npmjs.com/package/auto-ngtemplate-loader

Published a 3.1.0 with updated dependency versions for `html-loader`.
